docs/content/preview/drivers-orms/nodejs/yugabyte-node-driver.md

```diff
@@ -176,10 +184,9 @@ Employees Information:
 
 If there is no output or you get an error, verify the parameters included in the connection string.
 
-After completing these steps, you should have a working Node.js application that uses the YugabyteDB node-postgres smart driver to connect to your cluster, set up tables, run queries, and print out results.
-
 ## Learn more
 
-- Build Node.js applications using [Sequelize ORM](../sequelize).
-- [Node.js driver reference](../../../reference/drivers/nodejs/yugabyte-pg-reference/#fundamentals)
-- [YugabyteDB smart drivers for YSQL](../../smart-drivers/)
+- Refer to [YugabyteDB node-postgres smart driver reference](../../../reference/drivers/nodejs/yugabyte-pg-reference/) and [Try it out](../../../reference/drivers/nodejs/yugabyte-pg-reference/#try-it-out) for detailed smart driver examples.
```

Comment:
Ya I don't think the periods are called for. If bullets are a full sentence then yes, but otherwise no. We're probably not totally consistent about this.